configureWorkspace/configure.ts

```diff
@@ -593,7 +636,8 @@ export async function configure(): Promise<void> {
 
     await Promise.all(Object.keys(DOCKER_FILE_TYPES).map((fileName) => {
         // don't generate docker-compose files for .NET Core apps
-        if (platformType.toLowerCase().includes('.net') && !fileName.includes('docker-compose')) {
+        if (platformType.toLowerCase().includes('.net') && fileName.includes('docker-compose')) {
```

Comment:
FYI thanks for noticing!  Fix already [merged](https://github.com/Microsoft/vscode-docker/pull/296), even though you got to it sooner. :-)